inc/classes/class-service-worker.php

```diff
@@ -97,6 +98,97 @@ public function cache_images( \WP_Service_Worker_Scripts $scripts ) {
 
 	}
 
+	/**
+	 * Pre-Cache latest blog posts
+	 *
+	 * @param \WP_Service_Worker_Scripts $scripts Instance to register service worker behavior with.
+	 *
+	 * @return void
+	 */
+	public function precache_latest_blog_posts( \WP_Service_Worker_Scripts $scripts ) {
+
+		$recent_posts = new \WP_Query(
```

Comment:
@chandrapatel We might need caching here for `WP_Query` as it is going to run this query on every  page load.